frontend/smart-kickers-game/src/components/Heatmap/Heatmap.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import HeatMap from 'react-heatmap-grid';
+import './Heatmap.css';
+import { Colors } from './Colors';
+
+function Heatmap({ heatmap }) {
+  const heatmapDim = heatmap.length;
+  const array = new Array(heatmapDim).fill(0).map(() => '');
+  let numbersCopy = JSON.parse(JSON.stringify(heatmap));
+
+  for (let i = 0; i < heatmapDim; i++)
+    for (let j = 0; j <= i; j++) {
+      numbersCopy[i][j] = numbersCopy[i][j] + numbersCopy[j][i] - numbersCopy[j][i];
+      numbersCopy[j][i] = numbersCopy[i][j];
+    }
+
+  function chooseColor(value) {
+    let chosenColor = Colors.none;
+    if (value > 2 && value <= 5) {
+      chosenColor = Colors.blue;
+    } else if (value > 5 && value <= 10) {
```

Comment:
In every `else if` the first comparison is needless. Let's say `value` is equal 8. It doesn't go inside the first comparison because it's greater than 5, ok. But here `(value > 5 && value <= 10)` the comparison `value > 5` is needless because if it was it would go inside in the first one.

Reply:
Just a suggestion, consider using Switch to eliminate that huge amount of `else ifs`.

Reply:
Done. Inserted switch statement